invenio_rdm_records/services/errors.py

```diff
@@ -52,3 +52,44 @@ def __init__(self, *args, **kwargs):
             *args,
             **kwargs,
         )
+
+
+class ValidationErrorWithMessageAsList(Exception):
```

Comment:
Start here to understand the error handling motivation and see the test to see it in action.